### DISCUSSION TITLE
Monthly Roadmap Updates?  Any news?

### DISCUSSION THREAD

**Discussion by tamsky** [aws/aws-cli#7213] [General]
On March 8, 2021: @shaunhickson said:
https://github.com/aws/aws-cli/issues/4947#issuecomment-793192340 

> Hi all - I’m the manager of the AWS CLI team, ...
...
> Starting in April, we’re going to begin posting monthly roadmap updates as issues on our GitHub repository. These will lay out what we’re actively working on and what’s on the horizon. 

Curious if these "Roadmap updates" ever started?

**Comment by aBurmeseDev** [comment]
Hi @tamsky, thank you for reaching out!

Upon checking with the CLI team internally, PyPI work is not currently on the team’s roadmap and monthly updates are no longer planned as of now. And the manager (Shaun) who mentioned that left the team over a year ago.

However, there's [this tracking issue](https://github.com/aws/aws-cli/issues/6785) on general ongoing work related to installation and we plan to continue making periodic announcements in GitHub discussion under announcements: https://github.com/aws/aws-cli/discussions/categories/announcements

Please let me know if you have any other questions!

**Reply by tamsky** [reply]
Is there a public [v2] roadmap?

**Reply by aBurmeseDev** [reply]
We do not currently have public roadmaps for CLI v2 but will continue to add [announcements](https://github.com/aws/aws-cli/discussions/categories/announcements) periodically going forward.
